server/repositories/hubContent.js

```diff
@@ -18,7 +18,7 @@ const hubContentRepository = httpClient => {
     const endpoint = `${config.api.hubContent}/${id}`;
 
     if (!id) {
-      logger.error(`HubContentRepository (contentFor) - No ID passed`);
+      logger.info(`HubContentRepository (contentFor) - No ID passed`);
```

Comment:
reducing these to `info`. These currently spam the logs and only represent a linked entity not existing - e.g if there is no associated series with a piece of content. So not actually errors